R/epipredict-data.R

```diff
@@ -51,31 +55,70 @@
 #'
 #' Data set for a handful of states on masking and distancing behaviours
-#' during the COVID-19 Pandemic and downloaded from the CMU Delphi COVIDcast
-#' Epidata API. This data set covers the period from
-#' June to December 2021.
+#' during the COVID-19 Pandemic, and downloaded from the CMU Delphi COVIDcast
+#' Epidata API. This data set covers the period from June to December 2021,
+#' and is limited to California, Florida, Texas, New Jersey, and New York.
+#'
+#' @format An object of class \code{tbl_df} (inherits from \code{tbl}, \code{data.frame}) with 1055 rows and 4 columns.
+#' @section Data dictionary:
+#' A tibble with columns:
+#' \describe{
+#'   \item{geo_value}{the geographic value associated with each row
+#'       of measurements.}
+#'   \item{time_value}{the time value associated with each row of measurements.}
+#'   \item{masking}{Estimated percentage of people who wore a mask for most or all of the time while in public in the past 7 days; those not in public in the past 7 days are not counted.}
+#'   \item{distancting}{Estimated percentage of respondents who reported that all or most people they enountered in public in the past 7 days maintained a distance of at least 6 feet. Respondents who said that they have not been in public for the past 7 days are excluded.}
+#' }
+#'
+#' @source Data from Delphi COVIDcast. Obtained via the Delphi Epidata API.
+#'   \url{https://cmu-delphi.github.io/delphi-epidata/api/covidcast.html}. This
+#'   object contains a modified part of the
+#'   \href{https://cmu-delphi.github.io/delphi-epidata/symptom-survey/#covid-19-trends-and-impact-survey}{data
+#'   aggregations in the API} that are prepared from the
+#'   \href{https://www.pnas.org/doi/full/10.1073/pnas.2111454118}{COVID-19
+#'   Trends and Impact Survey}; see the first link for more information on
+#'   citing in publications. These aggregations are licensed under the terms of
+#'   the \href{https://creativecommons.org/licenses/by/4.0/}{Creative Commons
+#'   Attribution license}.
+#'
+#' Modifications:
+#' * Selected a small subset of the available data.
+#' * Changed to wide format.
 "ctis_covid_behaviours"
 
-#' COVID-19 Incident Cases and Deaths
+#' Subset of COVID-19 incident cases and deaths from 5 states
 #'
 #' Data set for 5 states containing COVID-19 Incident Cases and Deaths as
-#' reported
-#' by JHU-CSSE and downloaded from the CMU Delphi COVIDcast Epidata API.
-#' This data set covers the period from June 2021 to December 2021, and is
-#' used in the epipredict Vignette on ... .
+#' reported by JHU-CSSE and downloaded from the CMU Delphi COVIDcast Epidata
+#' API. This data set covers the period from June 2021 to December 2021, and
+#' is limited to California, Florida, Texas, New Jersey, and New York.
+#'
+#' @format An object of class \code{epi_df} (inherits from \code{tbl_df}, \code{tbl}, \code{data.frame}) with 1055 rows and 4 columns.
+#' @section Data dictionary:
+#' A tibble with columns:
```

Comment:
thought: maybe it'd be more convenient to prepare some of this format stuff (class, row counts, column counts) with some helper functions +
```
#' `r <expression>`
```
stuff.  It might trade off between different types of typos&errors we could do, not sure on this aspect.

Reply:
RE automatic format generation, will leave this as-is for now since we don't intend to update the package frequently. We shouldn't need to add much new format info.